lib/network/options.js

```diff
@@ -598,9 +599,10 @@ let configureOptions = {
     },
     smooth: {
       enabled: true,
-      type: ['dynamic', 'continuous', 'discrete', 'diagonalCross', 'straightCross', 'horizontal', 'vertical', 'curvedCW', 'curvedCCW', 'cubicBezier'],
+      type: ['dynamic', 'continuous', 'discrete', 'diagonalCross', 'straightCross', 'horizontal', 'vertical', 'curvedCW', 'curvedCCW', 'cubicBezier', 'arc'],
       forceDirection: ['horizontal', 'vertical', 'none'],
-      roundness: [0.5, 0, 1, 0.05]
+      roundness: [0.5, 0, 1, 0.05],
+      radius: [0,100,200,400],
```

Comment:
The radius is `[default, min, max, step]`. Having the default outside the range and the step four times the range renders this unusable.

Reply:
Thats the thing as similar. The difference is just that i want to provide angle / length to get results as above in the image. Thats the part i am a bit struggling with.

Reply:
Okay.

What you want to play with is `Math.atan2(dy, dx)`. This will give you the angle from one node to the other. But you also have to compute `dy` and `dx` the same way. Computing one as from - to and the other as to - from is just not going to work.